### PR TITLE
Fixes handling of while loop during pass

### LINKED ISSUE #1569: Incorrect result of print inside do loop
Reference - https://github.com/lfortran/lfortran/issues/1395 
```
def f():                                            # Equivalent python code of #1395 lfortran
    j :i32 = 0
    while True:
        j = j + 1
        print(j)
        if(j>4):
            break
    print("out of outer loop")
f()

(lp) C:\Users\kunni\lpython>python try.py
1
2
3
4
5
out of outer loop

(lp) C:\Users\kunni\lpython>src\bin\lpython try.py
1
2
3
4
5
5
5
out of outer loop
```
I think https://github.com/lfortran/lfortran/pull/1401 is the correct direction for a similar fix here .

### PR DESCRIPTION
**Pull request by anutosh491**
Fixes #1569 
```
def f():                                            
    j :i32 = 0
    while True:
        j = j + 1
        print(j)
        if(j>4):
            break
    print("out of outer loop")
f()
```
On Branch
```
(lp) C:\Users\kunni\lpython>lpython try.py
1
2
3
4
5
out of outer loop
```
On Master
```
(lp) C:\Users\kunni\lpython>src\bin\lpython try.py
1
2
3
4
5
5
5
out of outer loop
```

### CODE REVIEW

File: integration_tests/loop_05.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+def f():
+    j :i32 = 0
+    while True:
+        j = j + 1
+        print(j)
+        if(j>4):
```
Comment (czgdp1807) [inline integration_tests/loop_05.py:6]:
```suggestion
        if j > 4:
```

File: integration_tests/loop_05.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+            break
+    print("out of outer loop")
```
Comment (czgdp1807) [inline integration_tests/loop_05.py:8]:
```suggestion
    print("out of outer loop")
    assert j == 5
```

**Review by czgdp1807 — CHANGES_REQUESTED** [review]

File: integration_tests/CMakeLists.txt [hunk 3]
```diff
@@ -258,6 +258,7 @@ RUN(NAME loop_01             LABELS cpython llvm c)
 RUN(NAME loop_02             LABELS cpython llvm c wasm wasm_x86 wasm_x64)
 RUN(NAME loop_03             LABELS cpython llvm c wasm wasm_x64)
 RUN(NAME loop_04             LABELS cpython llvm)
+RUN(NAME loop_05             LABELS cpython llvm)
```
Comment (czgdp1807) [inline integration_tests/CMakeLists.txt:261]: Works for `c` backend as well?

Comment (anutosh491) [reply integration_tests/CMakeLists.txt:261]: Yeah should work !

**Review by czgdp1807 — APPROVED** [review]

**Pull request merged**